app/src/test/java/com/duckduckgo/app/browser/applinks/DuckDuckGoAppLinksHandlerTest.kt

```diff
@@ -38,8 +38,8 @@ class DuckDuckGoAppLinksHandlerTest {
 
     @Before
     fun setup() {
-        whenever(appBuildConfig.sdkInt).thenReturn(Build.VERSION_CODES.N)
-        testee = DuckDuckGoAppLinksHandler(appBuildConfig)
+        whenever(appBuildConfig.sdkInt).thenReturn(Build.VERSION_CODES.O)
```

Comment:
nit: Since `DuckDuckGoAppLinksHandler` no longer depends on `appBuildConfig`, that mock can be removed entirely.